09_elements.html

```diff
@@ -289,4 +289,202 @@ <h2 id="element-location-strategies">Element Location Strategies</h2>
+     run the following substeps:
+
+     <ol>
+      <li>Apply the <a>element displayed</a> algorithm to <var>parent</var>.
+
+      <li>If the return value is false, abort these steps and return that value.
+
+      <li>If <var>parent</var> is a block element box and the
+       computed values of either <code>overflow-x</code>
+       or <code>overflow-y</code> is "hidden":
+
+      <ol>
+       <li>Let <var>parent dimensions</var> be the <code>DOMRect</code>
+        that is the first element of the <code>DOMRectList</code>
+        array returned by calling
+        <code><a href=http://www.w3.org/TR/cssom-view/#dom-range-getclientrects>getClientRects</a></code>
+        on <var>parent</var>.
+
+       <li>Let <var>element dimensions</var> be the <code>DOMRect</code>
+        that is the first element of the <code>DOMRectList</code>
+        array returned by calling
+        <code><a href=http://www.w3.org/TR/cssom-view/#dom-range-getclientrects>getClientRects</a></code>
+        on <var>element</var>.
+
+       <li>Let <var>parent style</var> be the computed style of <var>parent</var>.
+
+       <li>Return false if any the following conditions evaluate to false:
+
+       <ul>
+        <li><var>element dimension</var>'s <code>top</code>
+         is less than (<var>parent dimension</var>'s <code>bottom</code>
+         − the <a title="normalize style pixel values to floating point">normalized style pixel float value</a>
+         of <var>parent style</var>'s <code>borderBottomWidth</code>).
+
+        <li><var>element dimension</var>'s <code>bottom</code>
+         is less than (<var>parent dimension</var>'s <code>top</code>
+         − the <a title="normalize style pixel values to floating point">normalized style pixel float value</a>
+         of <var>parent style</var>'s <code>borderTopWidth</code>).
+
+        <li><var>element dimension</var>'s <code>left</code>
+         is less than (<var>parent dimension</var>'s <code>right</code>
+         − the <a title="normalize style pixel values to floating point">normalized style pixel float value</a>
+         of <var>parent style</var>'s <code>borderRightWidth</code> casted as a float).
+
+        <li><var>element dimension</var>'s <code>right</code>
+         is less than (<var>parent dimension</var>'s <code>left</code>
+         − the <a title="normalize style pixel values to floating point">normalized style pixel float value</a>
+         of <var>parent style</var>'s <code>borderLeftWidth</code> casted as a float).
+      </ul>
+
+      <li>Run step 10 on the parent elements of <var>parent</var>, if any.
+     </ol>
+    </ol>
+
+    <li>Return true.
+   </ol>
+
+   <p>If the steps above return true, then the element MUST also
+    be <a href=#interactable-elements>interactable</a> should it
+    meet the other criteria for being interactable.
+    If any part of the <code>BODY</code> can be brought into the current viewport,
+    the return value MUST be true.
```

Comment:
I honestly don't know, it was there from before I made my changes.  I think it's probably wrong.